src/reporters/GriddedDataReporter.C

```diff
@@ -0,0 +1,39 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "GriddedDataReporter.h"
+#include "DelimitedFileReader.h"
+#include "GriddedData.h"
+
+registerMooseObject("isopodApp", GriddedDataReporter);
+
+InputParameters
+GriddedDataReporter::validParams()
+{
+  InputParameters params = GeneralReporter::validParams();
+  params.addClassDescription(
+      "Reporter to hold griddedData parameter values and their spatial and temporal coordinates");
```

Comment:
params.addClassDescription(
      "Reporter to hold griddedData parameter values and a grid containing their spatial and temporal coordinates");

Reply:
done